301-web-app-regional-vnet-private-endpoint-sql-storage/azuredeploy.json

```diff
@@ -0,0 +1,458 @@
+            "type": "array",
+            "metadata": {
+                "description": "JSON object describing virtual networks & subnets"
+            }
+        }
+    },
+    "variables": {
+        "suffix": "[substring(replace(guid(resourceGroup().id), '-', ''), 0, 6)]",
+        "appName": "[concat(parameters('webAppName'), '-', variables('suffix'))]",
+        "storagePrivateDnsZoneName": "privatelink.blob.core.windows.net",
+        "sqlPrivateDnsZoneName": "privatelink.database.windows.net",
+        "sqlDatabaseName": "mydb01",
+        "storageContainerName": "mycontainer",
+        "storageGroupType": "blob", // valid values: 'blob', 'file', 'table', 'queue', 'web', 'dfs'
+        "sqlGroupType": "sqlServer", // valid values: 'sqlServer'
+        "vnetNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/vnets.json', parameters('_artifactsLocationSasToken')))]",
+        "vnetPeeringNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/vnet_peering.json', parameters('_artifactsLocationSasToken')))]",
+        "appServicePlanNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/app_svc_plan.json', parameters('_artifactsLocationSasToken')))]",
+        "appNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/app.json', parameters('_artifactsLocationSasToken')))]",
+        "sqlNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/sqldb.json', parameters('_artifactsLocationSasToken')))]",
+        "privateLinkNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/private_link.json', parameters('_artifactsLocationSasToken')))]",
+        "storageNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/storage.json', parameters('_artifactsLocationSasToken')))]",
+        "privateDnsNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/private_dns.json', parameters('_artifactsLocationSasToken')))]",
+        "privateDnsRecordNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/dns_record.json', parameters('_artifactsLocationSasToken')))]",
+        "privateLinkIpConfigsNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/private_link_ipconfigs.json', parameters('_artifactsLocationSasToken')))]",
+        "privateLinkIpConfigsHelperNestedTemplateUri": "[uri(parameters('_artifactsLocation'), concat('nestedtemplates/private_link_ipconfigs_helper.json', parameters('_artifactsLocationSasToken')))]"
+
+    },
+    "resources": [
+        {
+            "comments": "This deployment links to the vnets.json nested ARM template",
+            "name": "[concat('linkedTemplate-vnet', '-', copyIndex())]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2018-05-01",
+            "resourceGroup": "[resourceGroup().name]",

```

Comment:
remove this property

Reply:
Hi Brian, not sure what I need to do here. Which property are you referring to?